ci-operator/step-registry/openshift-qe/perfscale-ibm-data-path-sg/openshift-qe-perfscale-ibm-data-path-sg-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# Configure ibmcloud
+# to find way to store region and ic_api_key
+export IBMCLOUD_CLI=ibmcloud
+export IBMCLOUD_HOME=/output
+region=$(jq -r .ibmcloud.region ${SHARED_DIR}/metadata.json)
+export region
+"${IBMCLOUD_CLI}" config --check-version=false
+
+echo "logging in..."
+"${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+"${IBMCLOUD_CLI}" plugin list
+resource_group_name=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
+echo resource group is $resource_group_name
+
+ibmcloud resource group $resource_group_name || exit 1
+ibmcloud target -g $resource_group_name -r $region
+sg=$(ibmcloud is sgs --resource-group-name $resource_group_name --output json | jq -r '.[] | select(.name | contains("cluster-wide"))|.id')
+echo "Print resource group in detail"
+
+ibmcloud is sgs --resource-group-name $resource_group_name
```

Comment:
@SachinNinganure  You have got the sg above in line 25. Is this line just to output for debugging?

Reply:
line 25 is to fetch the security group with the name cluster-wide (non-default, due to this the code was stuck and long pending to be running) which I will be using to add the rules at line 37 and 38, while line 28 will display the present security groups of the cluster. thank you for the question